whichone/static/js/main.js

```diff
@@ -37,22 +38,31 @@ var refreshIntervalId = null;
 var ps = 0;
 
 $(document).ready(function () {
-  document.getElementById("mode_text").style.color = "#FFC789";
-  countdownNumberElement = document.getElementById("countdown-number");
-  countdownNumberElement.textContent = countdown;
   spotify_id = $("#info").data("user");
-
-  // if the current user has spotify data
-  // in the local storage, user that
-  // instead of requesting new data
-  if (!localDataFound()) {
-    getSpotifyData();
-  }
-
+  initData();
   initOptions();
   setMuteIcon();
 });
 
+async function initData(currentTime = Date.now()) {
```

Comment:
Better to inject a factory than a optional param.

You are assuming currentTime will be used immediately but better to not make that assumption :)